docs/advanced-usage/uninstallation.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Uninstallation
+description: ⚡ An advanced view on how to uninstall Spicetify.
+---
+
+## Windows
+
+### Command Prompt (Administrator) - Spicetify Removal
+```cmd
+spicetify restore
+rmdir %userprofile%\.spicetify\ /S
+rmdir %userprofile%\spicetify-cli\ /S
+```
+
+### Powershell (Administrator) - Optional Path Removal 
+```powershell
+$path = [System.Environment]::GetEnvironmentVariable("PATH", "User")
+$replacement = ([System.Environment]::GetEnvironmentVariable("PATH", "User").Split(";") | Where-Object { $_.TrimEnd("") -ne "${HOME}\spicetify-cli" }) -join ";"
+if ("$path".Contains("${HOME}\spicetify-cli")) {[Environment]::SetEnvironmentVariable("PATH", "${replacement}", "User")}
+```
```

Comment:
Since the obligatory process is mostly surrounding removing the folders I'd remove the PATH removal since advanced users will already know how to do it. Maybe we can add support for this in the CLI actually.
```suggestion
```